src/Monitor/Monitor.Test/Autoscale/NewAzureRmAutoscaleRuleTests.cs

```diff
@@ -59,10 +59,11 @@ public void NewAutoscaleRuleCommandParametersProcessing()
             Cmdlet.TimeGrain = TimeSpan.FromMinutes(1);
             Cmdlet.ExecuteCmdlet();
 
-            Cmdlet.TimeWindow = TimeSpan.FromMinutes(4);
+            // TimeWindow minimum value is now 1 min. 
+            Cmdlet.TimeWindow = TimeSpan.FromMinutes(0);
```

Comment:
??

Reply:
Minimum TimeWindow is now 1 min.  If you pass over 0 min, we're expecting ArgOutOfRange thrown.

Reply:
In fact, passing 0 here is the same as not sending the argument (it is the default value).
This has to be changed to something like TimeSpan.FromSeconds(59). That would throw the ArgOutOfRangeException